### PR TITLE
Add Obsidian

### LINKED ISSUE #3731: Need icon of Obsidian.md
**Name:** obsidian
**Website:** https://obsidian.md/
**Alexa rank:** 75799
  

**Official resources for icon and color:**  icon

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Obsidian

**Issue:** Resolves #3731
**Alexa rank:** [~50.3k](https://www.alexa.com/siteinfo/obsidian.md)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon from SVG in [source](https://obsidian.md/) header, using a `0.35dp` cut-out. Colour from website stylesheet. Fiddlier than it first seemed, I had to do a fair bit of manual adjustment. To help with the comparison to the original, try changing the coordinates & dimensions of the path to:
- X: 4.440
- Y: 0.091
- W: 15.100
- H: 23.838

### CODE REVIEW

**Comment by adamrusted** [comment]
Just FYI - the coordinates you gave to compare end up in [this mess](https://user-images.githubusercontent.com/1830867/112963909-bde80a00-913f-11eb-8868-99c7f5c9f214.png)!
Just going to rebuild the icon myself with .35dp gaps, see if I can get that to match up for comparison.

**Comment by PeterShaggyNoble** [comment]
😕 Dunno what happened with those coordinates, maybe I simply posted the wrong ones, from before I'd finished fiddling with it. Or I may have inadvertently pushed one of my work in progress files.

Took another stab at it from scratch and with a slightly different approach which didn't need nearly as much manual adjustment afterwards. Coordinates to help with comparison this time are: `x = 4.439`, `y = 0.053`, `w = 15.104` & `h = 23.907`

**Comment by adamrusted** [comment]
Still seeing [differences](https://user-images.githubusercontent.com/1830867/114884105-67e0bb00-9dfd-11eb-8330-6e6bf269e523.png) when using those coordinates 🤔, and [different inconsistencies](https://user-images.githubusercontent.com/1830867/114884300-92cb0f00-9dfd-11eb-96fe-e6233c6cb6d2.png) when I ignore the coordinates 😅 - could another @simple-icons/maintainers take a look at this one, 'cause I don't know if it's just me or my software going mad!

**Comment by PeterShaggyNoble** [comment]
I'd say it's most likely me! This one's giving me _far_ too much trouble for such a simple icon!

**Comment by ericcornelissen** [comment]
I'm getting pretty good results **if** I set the width of the two SVGs equal to each other. Yes, the original SVG becomes to tall, but the rest of the SVGs line up [nearly perfectly](https://user-images.githubusercontent.com/3742559/126041187-b491d05f-39f9-4b42-8cd8-2e420c54a948.png). I'd say this makes sense given that parts of the border were omitted. What do others think about this?

**Comment by PeterShaggyNoble** [comment]
If there are still issues with this one, I'm happy to bow out and let someone else have a crack at it. Such a simple looking icon shouldn't cause so many issues! 😆

**Comment by roaldarbol** [comment]
Would be super stoked to see this one through. Would it be possible to merge the this icon and then leave it open for future improvements? I'd be happy to help, but working with SVGs is well outside my comfort zone unfortunately.

**Comment by ericcornelissen** [comment]
Thank you for pinging @roaldarbol. We generally want to avoid including icons that are too low in quality because they risk not being updated, because later never comes.

That said, per [my previous comment](https://github.com/simple-icons/simple-icons/pull/5287#issuecomment-881912581) I think the SVG currently found in this Pull Request lines up well enough with the original that, without a precedent on this style of icon, I'd be fine to merge it. However, it's been a while since I reviewed icons for this project, so I'd prefer it if another @simple-icons/maintainers could have a look at this to see if they agree. (if they disagree, per [Peter's comment](https://github.com/simple-icons/simple-icons/pull/5287#issuecomment-898481441), this can be closed so someone else can pick up this icon).

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6573,6 +6573,11 @@
             "hex": "353E58",
             "source": "https://observablehq.com/"
         },
+        {
+            "title": "Obsidian",
+            "hex": "483699",
```
Comment (sachinraja) [inline _data/simple-icons.json:6578]: Everything looks good except for the hex color here. I am not seeing this color anywhere on their website. I would go with `#7f6df2` which is under the CSS variable `--text-accent` on their website.

**Review by sachinraja — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (sachinraja) [reply _data/simple-icons.json:6578]: Oh, nevermind I found it. It's the CSS variable `--interactive-accent`.

**Review by sachinraja — APPROVED** [review]
LGTM @ericcornelissen

**Pull request merged**